std_msgs/Header header

# Index 0 is sensor 0, index 1 is sensor 1
int32[2] pressure # In PSI
int32[2] position # In % of full stroke
